e2e-tests/fixtures/Plan.ts

```diff
@@ -94,12 +94,31 @@ export class Plan {
 
   async addActivity(name: string = 'GrowBanana') {
     const currentNumOfActivitiesWithName = await this.panelActivityDirectivesTable.getByRole('row', { name }).count();
-    await this.page.getByRole('button', { name: `CreateActivity-${name}` }).click();
+    const activityListItem = this.page.locator(`.list-item :text-is("${name}")`);
+    const activityRow = this.page.locator('.timeline .rows .timeline-row-wrapper:first-of-type .overlay');
```

Comment:
Can we get the activity row by finding the name of the row? Something like `locator('.timeline').getByRole('listitem').filter({ hasText: 'Activities by Type' })` might work.